### ISSUE TITLE
Changing Domain name

### ISSUE THREAD

**Issue by guymass** [WWBN/AVideo#7928]
I want to change a domain name for my test site, do I need to make changes in the script to accept this change, and where do I do that?

**Comment by DanielnetoDotCom** [comment]
https://github.com/WWBN/AVideo/wiki/Change-the-URL-of-your-installation

**Comment by guymass** [comment]
Half way changing my domain I remembered it has few more services that needs that DNS to remain, so I  changed it back. I was only changing the virtual host files and then I changed them back. Now whenI try to reopen my test site fetube it shows me the following page:

![Screen Shot 5783-07-30 at 22 46 42](https://user-images.githubusercontent.com/58882903/233722366-6bd4a8e5-491c-4735-893e-6e0bb476e48e.jpg)


The DNS are already back and I  only change the ServerName in my config, nothing special it didn't take 5min to change back. So why now it my site showing this White Page? I don't think I made any special changes more then DNS and even that I only changed for the main domain and not the subdomain which is already pointing to my site.
